src/prommis/uky/uky_flowsheet.py

```diff
@@ -212,32 +214,30 @@ def main():
     Run the flowsheet by calling the appropriate functions in series.
     """
     m = build()
+    set_partition_coefficients(m)
 
     set_operating_conditions(m)
 
     scaled_model = set_scaling(m)
+
     assert_units_consistent(scaled_model)
```

Comment:
Bare assertions are not good in user facing code. You should replace these with `if` statements that raise an appropriate error with a useful error message.